americangut/per_category.py

```diff
@@ -26,11 +28,83 @@ def cat_taxa_summaries():
         for col in table.ids():
             if col == 'Unknown':
                 continue
-            cleaned_col = col.split('(')[0].strip().replace(' ', '_')
+            # Clean the string to just the base information of the response
+            # and url compliant names.
+            cleaned_col = col.split('(')[0].strip().replace(
+                ' ', '_').rstrip('+')
             filename = '-'.join([name, cleaned_col]) + '.txt'
 
             with open(join(out_dir, filename), 'w') as f:
                 for otu, val in zip(ids, table.data(col)):
                     if val == 0.0:
                         continue
                     f.write('%s\t%s\n' % (otu, str(val)))
+
+
+def cat_alpha_plots():
```

Comment:
Hmm how about computing the md5 (ex. ```self.assertFileMD5()``` in [this](https://grass.osgeo.org/grass72/manuals/libpython/gunittest.html) package)? Otherwise simply verifying your output file (shannon_path, pd_path) exists as you expect?